guides/V1_wwsg_EditingMessages.md

```diff
@@ -9,15 +9,28 @@ Watson Work Services provides an **updateMessage** API for callers to change the
 
 ![Image](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/images/editMessage.png)
 
-Note: Only the author of a message can update its content.  Updating the content of messages not created by the calling user will receive a 403 Forbidden response.
+Notes:
+
+
+ - Only the author of a message can update its content.  Updating the content of messages not created by the calling user will receive a **403 Forbidden** response.
+
+ - Existing annotations will be stripped from an updated message.  Apps are expected to regenerate any annotation using the new content.  Mention annotations are automatically regenerated based on the new content.  These annotations will be sent as `message-annotation-added` events.
+
+ - Apps calling the updateMessage mutation using the app's identity will be rejected with a **400 Bad Request** response, but using a user's identity, an app will be allowed to update the user's messages as that user.
```

Comment:
should be

Apps calling the updateMessage mutation using the app's identity will be rejected with a 400 Bad Request response, but when the app is running on behalf of a user it will be allowed to update the user's messages as that user.